shell/plugins/dashboard-store/mutations.js

```diff
@@ -51,24 +51,16 @@ export function load(state, { data, ctx, existing }) {
 
   let entry;
 
-  function replace(existing, data) {
-    data = getters.cleanResource(existing, data);
-
-    for ( const k of Object.keys(existing) ) {
-      delete existing[k];
-    }
-
-    for ( const k of Object.keys(data) ) {
-      Vue.set(existing, k, data[k]);
-    }
+  function replace(existing, data, list) {
+    Vue.set(list, list.findIndex(item => item.id === existing.id), data);
 
     return existing;
   }
 
   if ( existing && !existing.id ) {
     // A specific proxy instance to used was passed in (for create -> save),
     // use it instead of making a new proxy
-    entry = replace(existing, data);
+    entry = classify(ctx, data);
```

Comment:
doesn't the comment above insinuate that the data object is already a model? if not where/how did this get `classify`d before?

Reply:
It "gets" classified on `replace` by reusing the existing object, so technically we never apply a direct `classify` to `data`.